numpy/core/_asarray.py

```diff
@@ -105,6 +122,16 @@ def asanyarray(a, dtype=None, order=None):
         'A' (any) means 'F' if `a` is Fortran contiguous, 'C' otherwise
         'K' (keep) preserve input order
         Defaults to 'C'.
+    like : array_like
```

Comment:
Can we do some docstring templating here with `array_function_dispatch`?

Reply:
I'm not exactly sure what do you mean by that, note that `keyword_name : array_like` is what's used throughout the documentation whenever `array_function_dispatch` is used, such as in https://github.com/numpy/numpy/blob/eec0aa278e3e0d68d9e19ccba2410c09be20d746/numpy/core/numeric.py#L82 .

Reply:
I mean write the text of this argument docstring in just one place, and replace occurrences of `$LIKE_ARG` or similar in the decorator.

Reply:
Interesting, I didn't know this was allowed, thanks for the suggestion. I'll do that together with the rephrasing.